src/grandorgue/gui/GOGUISequencerPanel.cpp

```diff
@@ -39,106 +43,152 @@ GOGUIPanel *GOGUISequencerPanel::CreateSequencerPanel(GOConfigReader &cfg) {
 
   GOGUILabel *PosDisplay
     = new GOGUILabel(panel, m_OrganController->GetLabel(wxT("Label")));
-  PosDisplay->Init(cfg, wxT("SetterCurrentPosition"), 350, 10);
+  PosDisplay->Init(cfg, wxT("SetterCurrentPosition"), 325, 10);
+  PosDisplay->SetFontSize(14);
   panel->AddControl(PosDisplay);
 
+  GOGUILabel *const currFileNameDisplay = new GOGUILabel(
+    panel, m_OrganController->GetLabel(GOSetter::KEY_CURR_FILE_NAME));
+
+  currFileNameDisplay->Init(
+    cfg, GOSetter::GROUP_CURR_FILE_NAME, 170, 58, wxEmptyString, 15, 380, 40);
+  currFileNameDisplay->SetFontSize(14);
+  /*
```

Comment:
Perhaps remove the  comments that contain dead code?

Reply:
Fixed